Jenkinsfile

```diff
@@ -148,11 +149,15 @@ stage('Prepare') {
   }
 }
 
-stage("Sanity Check") {
+stage('Sanity Check') {
   timeout(time: max_time, unit: 'MINUTES') {
     node('CPU') {
-      ws(per_exec_ws("tvm/sanity")) {
+      ws(per_exec_ws('tvm/sanity')) {
         init_git()
+        is_docs_only_build = sh (returnStatus: true, script: '''
```

Comment:
ok uhh sorry there is just one minor problem here: the polarity is backwards on this one so none of the if checks below make sense. can you fix?

Reply:
Good catch! This is fixed now! PTAL